gsoc/2020/mentors.md

```diff
@@ -17,3 +17,6 @@ layout: plain
 * Jim Pivarski [pivarski@princeton.edu](mailto:pivarski@princeton.edu) Princeton University
 * Alex Richards [a.richards@imperial.ac.uk](mailto:a.richards@imperial.ac.uk) Imperial College London
 * Mark Smith [mark.smith1@imperial.ac.uk](mailto:mark.smith1@imperial.ac.uk) Imperial College London
+* Jakob Blomer [jakob.blomer@cern.ch](mailto:jakob.blomer@cern.ch) CERN
```

Comment:
You need to rebase on master after you do the remaining fixes, then fix the conflicts in mentors.md and insert the new mentor names sorted agains the last names

Reply:
... sort against the other existing names in the list, not just the mentors of this project